### PR TITLE
fix(navigation): fixed scroll on navigation

### LINKED ISSUE #263: Page is scrolling longer than the navigation
If the content of the page is longer than the navigation you can scroll past the navigation when it's open on mobile.

### PR DESCRIPTION
**Pull request by frontendherodk**
Fixes #263

### CODE REVIEW

File: src/app/navigation.scss [hunk 1]
```diff
@@ -29,6 +29,10 @@
     display: flex;
     flex-wrap: wrap;
     z-index: 2;
+    .overlay-open & {
```
Comment (dami-gg) [inline src/app/navigation.scss]: Please move this out this .scn block, to a new block just for .overlay-open .scn It makes it easier to read it ;)

Comment (dami-gg) [inline src/app/navigation.scss]: Even better, move it inside the block at the end of this css file where there's already code for body.overlay-open

Comment (frontendherodk) [inline src/app/navigation.scss]: I don't agree with the readability. The style is connected to `.navigation` thus making more sense that it stays there.

Which part is not readable?

**Comment by dami-gg** [comment]
:+1:

**Comment by frontendherodk** [comment]
👍

**Pull request merged**